src/database/database_writer.py

```diff
@@ -0,0 +1,16 @@
+from .database_manager import DatabaseObjectManager
+
+
+class Writer:
+    def __init__(self, db_name, coll_name):
+        self.db_manager = DatabaseObjectManager()
+        self.db = self.db_manager.getDatabase(db_name)
+        self.collection = self.db_manager.getCollection(db_name, coll_name)
+
+    def addDocument(self, record):
+        """ Add one record to specified collection, returns id of inserted object """
+        return self.collection.insert_one(record)
+
+    def addDataset(self, dataset):
+        """ As an input function takes list of objects and return list of their ids """
+        return self.collection.insert_many(dataset)
```

Comment:
Maybe we should have function, which get some table and save it to database and function for reading table. Saving table is basic action in our program. I will import data from file to pandas `DataFrame`, so if you don't implement this, I will probably do it.

Reply:
Ok, i will add it asap. This funtion should take dataframe and table name as parameters or you want to pass something more?

Reply:
I need two functions. Both take dataframe. First create new document in database and save table. Second add dataframe to the existing document in database.

Reply:
Second function will be used to save one big file by dividing table into chunks.

Reply:
Ok, functions added. Important information:

- adding a dataframe is realised with modified function `def add_dataset(self, dataframe)` in Writer - now the input is a pandas DataFrame, mongo mechanisms deal with inserting in both cases - when collection exists or not
- reading chunk is implemented in Reader class: `def get_nth_chunk(self, query, columns, use_id=0, chunk_size=0, chunk_number=0)` - here, if both chunk_size and chunk_number stays 0, it will return all records that passed a query, in opposite - it will return chunk. **Chunks are indexed from 0**